src/components/dock/dock-button/dock-button.scss

```diff
@@ -6,7 +6,9 @@
     all: unset;
```

Comment:
This is from the example _Using CSS color variables for theming the Dock_, and the bright button (_Calls_) is being hovered.

(BTW, in all of these screenshots, when there are two copies side-by-side like this, unless I say anything else, it's the published docs for this PR on the left, and the latest published `next` on the right. I also check mostly in light mode, since that's where we primarily care about not breaking stuff.)

I assume it's not really a bug that the background colour is bright on hover, because it doesn't look like the hover colour is changed by the example, but I wanted to flag it to be on the safe side.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212026523-ec3d9b3d-3309-4423-a83e-f9e365b7f16a.png">

Reply:
Oh nice catch. I'll fix this